src/Application.vala

```diff
@@ -180,7 +180,7 @@ public class Terminal.Application : Gtk.Application {
 
     protected override void startup () {
         base.startup ();
-        Hdy.init ();
+        Adw.init ();
```

Comment:
Need to Granite.init () first to avoid getting Adw styles and make sure we load Granite styles
```suggestion
        Granite.init ();
        Adw.init ();
```